internal/controller/util/misc.go

```diff
@@ -236,13 +239,13 @@ func CreateNamespaceIfNotExists(ctx context.Context, k8sClient client.Client, na
 }
 
 func IsCGEnabled(annotations map[string]string) bool {
-	if annotations == nil {
-		return false
-	}
-
 	return annotations[IsCGEnabledAnnotation] == "true"
 }
 
+func IsRBDEnabledForVolSyncReplication(annotations map[string]string) bool {
```

Comment:
Maybe rename to UseVolsyncForPVCProtection()?

Reply:
Changed to `IsPVCMarkedForVolSync`